pymtl3/passes/yosys/test/TranslationImport_closed_loop_dircted_test.py

```diff
@@ -34,7 +34,7 @@ def run_sim( _th ):
   try:
     _th.elaborate()
     _th.q.yosys_translate = True
-    _th.q.yosys_import = True
+    _th.q.yosys_import = ImportConfigs()
```

Comment:
What is the purpose of instantiating ImportConfigs() here?

Reply:
Most of the import configurations implemented in this PR should go with the component to be imported. That's why I find it natural to reuse the `sverilog_import` attribute from which the import pass extracts configurations.

Reply:
My question was, since you probably need to specify import configs in the Placeholder, why do you still need to do this at the top level? Are you doing this because translation pass doesn't handle import?

Reply:
I think sometimes you still need to specify configs at top level. It's true that most Placeholders to be imported should have such configs by default, but these DUT's actually come from `stdlib` which certainly should not come with preset configs.

Reply:
Let me clarify. I think in this case stdlib component is a valid pymtl component with valid logic. Only when you translate it using yosys backend do you need to import it back. I was asking whether translation pass has a default import config to help something but seems like translation pass is what it is and we need to supply another import config before we apply import pass. In the case of translate-import we actually know the file names and such, maybe after the translate pass you can add a default import config to the component? Or we have a high level translateimport pass that wraps what you do here.

Reply:
Ah I see. The default values of `ImportConfigs` are set to be used with translation-import scenarios. To perform a translation-import you simply use `ImportConfigs()` to mark some component as to be imported instead of writing up all the options.

Reply:
also i just noticed that there is a typo in the file name

Reply:
Maybe the translation pass adds a default import configs to the component?

Reply:
That sounds weird to me... I think ideally the translation pass does what it does and we provide default import configs that work for the most common cases (translation import).

Reply:
I mean if the user only uses translation-import in the most basic case we probably shouldn't bother them with import config.

Reply:
What do you think? I was basically thinking that we may add the default import config to the component in the translation pass so that the user can directly apply import pass.

Reply:
I think it should be a translation-import pass. I'll add that.